examples/reference/panes/HoloViews.ipynb

```diff
@@ -139,24 +414,91 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pn.panel(dmap, center=True, widget_location='right_bottom')"
+    "print(dmap_panel)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Lets try to change the name of the `FloatSlider`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "float_slider = hv_panel[1][0]\n",
+    "float_slider.name=\"Frequency (updated)\"\n",
+    "float_slider"
```

Comment:
When I change the value of the floatslider it raises an exception. I don't know why and need help.

![image](https://github.com/holoviz/panel/assets/42288570/4cfbe94c-8647-4321-9952-efcecd9cf327)

Reply:
This is because you changed the name of the widget (which is not allowed). I will force the name to be constant.